pyproject.toml

```
@@ -131,7 +131,7 @@ all_extras = [
     "scikit_posthocs>=0.6.5",
     "seaborn>=0.11.0",
     "seasonal",
-    "statsforecast>=0.5.2,<1.6.0; python_version < '3.11'",
+    "statsforecast>=0.5.2,<1.8.0; python_version < '3.11'",
```

```suggestion
    "statsforecast>=0.5.2,<1.7.0; python_version < '3.11'",
```

We can't allow an yet unreleased minor version.

---

Also, do we need this python 3.11 check?

---

this is now addressed